pages/[league]/team/[teamid].tsx

```diff
@@ -116,10 +118,31 @@ function TeamPage({
       <Container>
         {/* Data for this page that we can also do: Roster, Historical Stats, etc. */}
         <TableHeading>Skaters</TableHeading>
+        <DisplaySelectContainer role="tablist">
+          <DisplaySelectItem
+            onClick={() => setDisplay(() => 'stats')}
+            active={display === 'stats'}
+            tabIndex={0}
+            role="tab"
+            aria-selected={display === 'stats'}
+          >
+            Stats
+          </DisplaySelectItem>
+          <DisplaySelectItem
+            onClick={() => setDisplay(() => '')}
+            active={display === ''}
+            tabIndex={0}
+            role="tab"
+            aria-selected={display === ''}
+          >
+            Advanced Stats
+          </DisplaySelectItem>
+        </DisplaySelectContainer>
         <TableWrapper>
           {!isLoading && (
             <TableContainer>
-              <SkaterScoreTable data={getSkaters()} />
+              {display && (<SkaterScoreTable data={getSkaters()} />)}
+              {!display && (<SkaterAdvStatsTable data={getSkaters()} />)}
```

Comment:
Let's change these lines in favor of a ternary operator. for instance 
````js
{ display ? <Skaterstats /> : <SkaterAdvstats /> }
````

this would be a bit nicer

Reply:
Done. Thanks for showing the easier way to do it.